bundles/org.openhab.ui/web/src/components/config/controls/item-picker.vue

```diff
@@ -10,7 +10,8 @@
       <f7-button slot="media" icon-f7="list_bullet_indent" @click.native="pickFromModel" />
     </f7-list-item>
     <!-- for placeholder purposes before items are loaded -->
-    <f7-list-item link v-show="!ready" :title="title">
+    <f7-list-item link v-else title="Loading...">
+      <f7-preloader />
       <f7-button slot="media" icon-f7="list_bullet_indent" @click.native="pickFromModel" />
```

Comment:
while we're at it: without having read the code, I didn't realize the "list_bullet_indent"-icon actually was a button! 😮 
I even didn't really notice it, thought it's just some kind of menu icon. And I might not be the only one. What about giving the button a little of a "button styling", like a frame or some raised effect?

Reply:
I tried different things and none really looked pleasing in all 3 themes and pixel densities, it was either too bulky or out of place - I wanted the item picker to remain essentially a list item without anything out of place. So I agree it's not ideal, but once to realize by hovering over the icon to discover it's a button, then you've discovered that feature. Happy to see some more designs but I really tried some months ago.